exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = true;
```

Comment:
here too you should be using the comparison operator instead of hardcoding the boolean. Can you write an expression that evaluates to `true` using  the variable `htmlLevel` and a comparison operator?

Reply:
I made the changes. Thanks Ellie.